docs/docsite/rst/getting_started/index.rst

```diff
@@ -1,99 +1,127 @@
-.. _getting_started_index:
+Getting Started with Ansible in a Self-Contained Project Structure
+==================================================================
```

Comment:
I'd stick with the original title. The self-containt project structure is what you are doing here, but I don't see that it's useful as a page title.